src/osp/Resource/AssetImporter.cpp

```diff
@@ -121,9 +121,43 @@ void osp::AssetImporter::load_part(TinyGltfImporter& gltfImporter,
             continue;
         }
 
-        // TODO: more stuff
         PrototypeMachine machine;
         machine.m_type = std::move(type);
+
+        for (std::string key : value.Keys())
```

Comment:
Iterating the string by value? You sure?

Reply:
No I'm not :) I wrote this long before I knew what `std::string_view` was.

Reply:
I'd say just do

for(auto const& key : value.Keys())
{}

in this situation.

No need to type erase the string into a string_view.